tests/util.py

```diff
@@ -15,15 +18,19 @@ def create_system_tests_index_name(testrun_uid: str) -> str:
     return create_index_name(testrun_uid, "test-kb")
 
 
-def create_e2e_tests_index_name(testrun_uid: str) -> str:
-    return create_index_name(testrun_uid, "test-app")
+def create_e2e_tests_index_name(testrun_uid: str,
```

Comment:
See comment above.  
The whole point of namespaces, is that a single index can support multiple namespaces (including the "default" namespace).

Reply:
Yeah right fixed this also.